nemo/utils/exp_manager.py

```diff
@@ -100,10 +100,7 @@ class StepTimingParams:
 @dataclass
 class EMAParams:
     enable: Optional[bool] = False
-    evaluate_ema_weights_instead: Optional[bool] = False
     decay: Optional[float] = 0.999
-    apply_ema_every_n_steps: Optional[int] = 1
-    start_step: Optional[int] = 0
```

Comment:
We can support start step and apply_ema_every_n_steps by using the step count inside of an optimizer right? Or is  it dependent on the optimizer whether it maintains the counter?